### ISSUE TITLE
Use palette library instead rewriting hex colors in JSON file

### ISSUE THREAD

**Issue by asumbek** [catppuccin/diohub#1]
Self-explanatory, basically use palette library instead rewriting hex colors in JSON file.

**Comment by sgoudham** [comment]
Thanks for raising this after what I mentioned in the original issue. 

I should have caught this in my review (my bad!) but it looks like `bake`'s source is included fully in the repositories and actually licensed under `MPL 2.0`. This requires us to link the full MPL license text within the README. 

E.g.

```
## 💝 Thanks to

- [UrNightmaree](https://github.com/UrNightmaree)
- [hyperupcall](https://github.com/hyperupcall)'s [bake](bake) is subject to the terms of the Mozilla Public License, v. 2.0. You can obtain a copy of the MPL at https://www.mozilla.org/en-US/MPL/2.0/.
```

While we don't necessarily try to enforce tooling across our ports, we should **_really_** move away from bake as:
- The full source code is included which is [MPL 2.0](https://github.com/hyperupcall/bake/blob/main/LICENSE). This is more restrictive than MIT which Catppuccin favours as its default license unless [absolutely necessary](https://github.com/catppuccin/catppuccin/blob/main/docs/port-creation.md#licensing).
- Even with `bake`, bash is hard to maintain and parse at a glance.

I just wanted to confirm with you that you were thinking of removing `bake` as a dependency entirely with the inclusion of the palette library. In any case, until we move away from `bake`, **We need to update the README as outlined above**

**Comment by asumbek** [comment]
@sgoudham I've removed `bake` completely from the repo tree followed up with credit under "Thanks To" section.

Commit: 64d39af
